docker-compose.yml

```diff
@@ -138,7 +138,10 @@ services:
     depends_on:
       gateway:
         condition: "service_healthy"
-      httpbin:
+      # Doesn't work on aarch64 Macs
+      # httpbin:
```

Comment:
This is interesting.  I don't think I realized it didn't work on Mac Silicon.  Good to know.

Reply:
Looks like Kong forked the original repo to intentionally support both `amd64` and `arm64`
https://github.com/Kong/httpbin

Maybe we should switch to using that version?